<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>900</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="MainWidget">
   <widget class="QTextEdit" name="editor">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1201</width>
      <height>841</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft JhengHei</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="FileMenu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="NewAction"/>
    <addaction name="NewWindowAction"/>
    <addaction name="OpenAction"/>
    <addaction name="SaveAction"/>
    <addaction name="SaveAsAction"/>
    <addaction name="separator"/>
    <addaction name="PrintAction"/>
    <addaction name="SetupAction"/>
    <addaction name="ToggleAction"/>
    <addaction name="separator"/>
    <addaction name="ExitAction"/>
   </widget>
   <widget class="QMenu" name="EditMenu">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="FindAction"/>
    <addaction name="findAllAction"/>
    <addaction name="ReplaceAction"/>
   </widget>
   <widget class="QMenu" name="FormatMenu">
    <property name="title">
     <string>Format</string>
    </property>
    <widget class="QMenu" name="menuColor">
     <property name="title">
      <string>Color</string>
     </property>
     <addaction name="bgColorAction"/>
     <addaction name="textColorAction"/>
    </widget>
    <addaction name="fontMenu"/>
    <addaction name="menuColor"/>
   </widget>
   <widget class="QMenu" name="ViewMenu">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuZoom">
     <property name="title">
      <string>Zoom</string>
     </property>
     <addaction name="ZoomInAction"/>
     <addaction name="ZoomOutAction"/>
    </widget>
    <addaction name="menuZoom"/>
    <addaction name="ColorMenu_2"/>
   </widget>
   <widget class="QMenu" name="TextMenu">
    <property name="title">
     <string>Text</string>
    </property>
   </widget>
   <widget class="QMenu" name="HelpAction">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="helpAction"/>
   </widget>
   <widget class="QMenu" name="ThemeMenu">
    <property name="title">
     <string>Theme</string>
    </property>
    <widget class="QMenu" name="menuDark">
     <property name="title">
      <string>Dark</string>
     </property>
     <addaction name="darkTheme1"/>
     <addaction name="darkTheme2"/>
     <addaction name="darkTheme3"/>
     <addaction name="darkTheme4"/>
    </widget>
    <widget class="QMenu" name="menuLight">
     <property name="title">
      <string>Light</string>
     </property>
     <addaction name="lightTheme1"/>
     <addaction name="lightTheme2"/>
     <addaction name="lightTheme3"/>
     <addaction name="lightTheme4"/>
    </widget>
    <addaction name="menuDark"/>
    <addaction name="menuLight"/>
   </widget>
   <addaction name="FileMenu"/>
   <addaction name="EditMenu"/>
   <addaction name="FormatMenu"/>
   <addaction name="ViewMenu"/>
   <addaction name="TextMenu"/>
   <addaction name="ThemeMenu"/>
   <addaction name="HelpAction"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="OpenAction">
   <property name="text">
    <string>Open </string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="SaveAction">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="SaveAsAction">
   <property name="text">
    <string>Save As</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="PrintAction">
   <property name="text">
    <string>Print</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="ToggleAction">
   <property name="text">
    <string>Toggle</string>
   </property>
  </action>
  <action name="ExitAction">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionctlr_n">
   <property name="text">
    <string>ctlr+n</string>
   </property>
  </action>
  <action name="FindAction">
   <property name="text">
    <string>Find</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="findAllAction">
   <property name="text">
    <string>Find All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="findPrecAction">
   <property name="text">
    <string>Find Prev</string>
   </property>
  </action>
  <action name="FindNextAction">
   <property name="text">
    <string>Find Next</string>
   </property>
  </action>
  <action name="ReplaceAction">
   <property name="text">
    <string>Replace</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="fontMenu">
   <property name="text">
    <string>Font</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="ColorMenu_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Status bar</string>
   </property>
  </action>
  <action name="NewWindowAction">
   <property name="text">
    <string>New Window</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="SetupAction">
   <property name="text">
    <string>Page Setup</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="ZoomInAction">
   <property name="text">
    <string>Zoom in</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Up</string>
   </property>
  </action>
  <action name="ZoomOutAction">
   <property name="text">
    <string>Zoom out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Down</string>
   </property>
  </action>
  <action name="bgColorAction">
   <property name="text">
    <string>Background Color</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="textColorAction">
   <property name="text">
    <string>Text Color</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="NewAction">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="helpAction">
   <property name="text">
    <string>GetHelp</string>
   </property>
  </action>
  <action name="darkTheme1">
   <property name="text">
    <string>Dark Theme1</string>
   </property>
  </action>
  <action name="darkTheme2">
   <property name="text">
    <string>Dark Theme2</string>
   </property>
  </action>
  <action name="darkTheme3">
   <property name="text">
    <string>Dark Theme3</string>
   </property>
  </action>
  <action name="darkTheme4">
   <property name="text">
    <string>Dark Theme4</string>
   </property>
  </action>
  <action name="lightTheme1">
   <property name="text">
    <string>Light Theme1</string>
   </property>
  </action>
  <action name="lightTheme2">
   <property name="text">
    <string>Light Theme 2</string>
   </property>
  </action>
  <action name="lightTheme3">
   <property name="text">
    <string>Light Theme3</string>
   </property>
  </action>
  <action name="lightTheme4">
   <property name="text">
    <string>Light Theme4</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
